data/human/hails.txt

```diff
@@ -687,6 +713,8 @@ phrase "friendly civilian"
 		"Want my opinion? You should"
 		"Be like the best ship captains and"
 		"Don't forget to"
+		"Always"
```

Comment:
This can lead to combinations like "Always always install a ramscoop". That doesn't sound right to me.